### ISSUE TITLE
Cancel link doesn't have sufficient color contrast. 

### ISSUE THREAD

**Issue by mgifford** [gctools-outilsgc/gcpedia#136]
Path | .oo-ui-buttonElement-button[href$="User\:Mike"][rel="nofollow"] > .oo-ui-labelElement-label
-- | --
<span class="oo-ui-labelElement-label">Cancel</span>
Fix the following:Element has insufficient color contrast of 3.86 (foreground color: #dd3333, background color: #eaecf0, font size: 10.5pt (14px), font weight: bold). Expected contrast ratio of 4.5:1

<img width="691" alt="Screenshot with WAVE Toolbar" src="https://user-images.githubusercontent.com/116832/72949871-8de0f200-3d57-11ea-8493-33a71f6f7f09.png">
